frontend/src/core/model/cells.ts

```diff
@@ -94,4 +97,15 @@ export interface CellState {
 
   /** handle to access the underlying cell */
   ref: React.RefObject<CellHandle>;
+
+  /** cell configuration */
+  config: CellConfig;
+}
+
+export interface CellConfig {
+  /**
+   * If false, the cell will not be run automatically.
+   * Cannot be true, and instead will be set to null.
+   */
+  autoRun?: false | null;
```

Comment:
unused and cant currently change - just boiler plate to get the action/reducer in place